### PR TITLE
kubernetes: upgrade to 1.13.5

### PR DESCRIPTION
**Pull request by celskeggs**
Bump the kubernetes version to a newer major release. Bump other dependencies as necessary.

This is part of #351.

---

Checklist:

 - [x] I have split up this change into one or more appropriately-delineated commits.
 - [x] The first line of each commit is of the form "[component]: do something"
 - [x] I have written a complete, multi-line commit message for each commit.
 - [x] I have formatted any Go code that I have changed with gofmt.
 - [x] I have signed each commit with my GPG key.
 - [x] My changes have passed CI.
 - [x] I have built my changes locally, and tested that the changes work as expected.
 - [x] I have deployed a cluster incorporating my changes, and checked that it deploys successfully.
 - [x] I have assigned this issue to an appropriate reviewer. (Choose @celskeggs if you are not otherwise certain.)
 - [x] I consider my PR complete and ready to be merged without my further input, assuming that it passes CI and code review.

### CODE REVIEW

File: platform/WORKSPACE [hunk 1]
```diff
@@ -163,7 +163,7 @@ go_repository(
 
 go_repository(
     name = "io_k8s_kube_openapi",
-    commit = "0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803",
+    commit = "c59034cc13d587f5ef4e85ca0ade0c1866ae8e1d",
```
Comment (cryslith) [inline platform/WORKSPACE]: out of curiosity, why this commit?

Comment (celskeggs) [reply platform/WORKSPACE]: I grabbed them from the Godeps in the apimachinery, client-go, and api repositories.

File: platform/.bazelrc [hunk 2]
```diff
@@ -1 +1,2 @@
+startup --host_jvm_args=-Xmx4G
```
Comment (cryslith) [inline platform/.bazelrc]: is this the smallest order of magnitude which worked?

Comment (celskeggs) [reply platform/.bazelrc]: No, this was just a number I dumped into it. Do you want me to fiddle with it?

Comment (cryslith) [reply platform/.bazelrc]: No, should be fine because JVM doesn't commit this amount of memory

**Pull request merged**